src/client/components/Meal.js

```diff
@@ -1,119 +1,218 @@
+  }
+
+  const [numberOfGuests, setnumberOfGuests] = useState("");
+  const [phoneNumber, setPhoneNumber] = useState("");
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
 
   const handlePhoneNumberChange = (event) => setPhoneNumber(event.target.value);
   const handleNameChange = (event) => setName(event.target.value);
   const handleEmailChange = (event) => setEmail(event.target.value);
-const handleNumberOfGuests=(event) => setnumberOfGuests(event.target.value);
-  
-return(
-  <>
-  
-  <button onClick={() => onShowAlert('success')}>
-    testbbbbutton
-  </button>
-    <div>
-        <p>{meal.title}</p>
-        <p>{meal.description}</p>
-        
-    </div>
-    {getReservations(meal.id) &&
-    <div>
-      <h1>Submit a new meal!</h1>
-        <form onSubmit={handleSubmit}>
-          <label>
-            PhoneNumber:
-            <input type="text" value={phoneNumber} onChange={handlePhoneNumberChange} />
-          </label>
-          <br></br>
-          <label>
-            Name:
-            <input type="text" value={name} onChange={handleNameChange} />
-          </label>
-          <br />
-          <label>
-            Email:
-            <input type="text" value={email} onChange={handleEmailChange} />
-          </label>
-          <br />
-
-          <label>
-            number of guests:
-            <input type="text" value={numberOfGuests} onChange={handleNumberOfGuests} />
-          </label>
-          <br />
-
-          <button type="submit">Submit Meal</button>
-        </form>
-    </div>}
-
-    <Alert
-        header={'Header'}
-        btnText={'Close'}
+  const handleNumberOfGuests = (event) => setnumberOfGuests(event.target.value);
+
+  return (
+    <>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "row-reverse",
+          justifyContent: "space-around",
+        }}
+      >
+        <div>
+          <p
+            style={{
+              color: "#fdf8e8",
+              backgroundColor: "#ce8884",
+              margin: "20px",
+              fontSize: "40px",
+            }}
+          >
+            {meal.title}
+          </p>
+          <p
+            style={{
+              color: "#fdf8e8",
+              backgroundColor: "#ce8884",
+              margin: "20px",
+              fontSize: "20px",
+            }}
+          >
+            {meal.description}
+          </p>
+          <img
+            src={getMealPicture(meal.id)}
+            style={{
+              height: "50%",
+              width: "50%",
+              marginBottom: "50px",
+            }}
+          ></img>
+        </div>
+        {getReservations(meal.id) && (
+          <div>
+            <h1
+              style={{
+                fontSize: "40px",
+                color: "#fdf8e8",
+                backgroundColor: "#ce8884",
+                margin: "20px",
+              }}
+            >
+              Submit a new meal!
```

Comment:
This where a reservation is being made though right? Then the title should be something like 'Make a reservation'.
'Submit a new meal' makes me think there will be a form to fill in to create a new meal.